actions/service.go

```diff
@@ -280,31 +279,56 @@ func buildRunManifest(record graveler.HookRecord, tasks [][]*Task) runManifest {
 	return manifest
 }
 
-func (s *Service) UpdateCommitID(ctx context.Context, repositoryID string, runID string, commitID string) error {
+// UpdateCommitID assume record is a post event, we use the PreRunID to update the commit_id and save the run manifest again
+func (s *Service) UpdateCommitID(ctx context.Context, repositoryID string, storageNamespace string, runID string, commitID string) error {
+	if runID == "" || commitID == "" {
+		return nil
```

Comment:
Why isn't this an error?

Reply:
true, I'll verify just the run ID and let update commit ID to none if needed

Reply:
I assume the pre event completed in order to post event to trigger, so we must of run ID. if this will change in the future. Prefer to skip if no run ID was sent to UpdateCommitID